components/profile/layout/layout.tsx

```diff
@@ -34,50 +56,60 @@ export const ProfileLayout: React.FC<LayoutProps> = ({ children, footerData, wid
 
   const onRedirectCallback = useMemo(() => createRedirectCallback(dashboardPath), [dashboardPath]);
 
+  const CustomAuth0Provider = ({ children }: { children: ReactNode }) => {
```

Comment:
Good catch! Will change it

Reply:
Update: moving it outside caused a bug I couldn't figure out, so I left the declaration inside the component and added a comment there in case it causes problems in the future.